components/generic/SimilarItems.stories.js

```diff
@@ -0,0 +1,48 @@
+import { storiesOf } from '@storybook/vue';
+import StoryRouter from 'storybook-vue-router';
+import SimilarItems from './SimilarItems.vue';
+
+const i18n = {
+  locale: 'en',
+  messages: {
+    en: {}
+  }
+};
+
+storiesOf('Generic', module)
+  .addDecorator(StoryRouter())
+  .add('Similar items', () => ({
+    i18n,
+    components: { SimilarItems },
+    data() {
+      return {
+        items: [
```

Comment:
Thumbnails for the items in this context will always be 200px wide but at the source image's original aspect ratio, so will rarely actually be square.

In the storybook therefore we need to use both portrait and landscape images to be sure they display as intended in this component.

Also, let's reuse the images we have in .storybook/assets/img/

Reply:
We can still control them so that they will always be a square. Maybe good to speak to the designer to see what they say?

Reply:
Indeed, and that's what we need to do, and you have done here. :)

I was just noting that we want to reflect that in the Storybook by using images that are not in fact square, but display as such.